docs/configuration.md

```diff
@@ -221,14 +221,14 @@ blockrelay:
   log-results: true
   # builder-configs contain specific configurations for different builders, with each builder defined by its public key.
   # The base score for each bid is the value to the proposer, in wei.  The final score is calculated by adding
-  # the 'offset' value for the specific builder, and then multiplying it by the 'factor' value.  For example,
-  # if the base value is 1000, the offset is 10 and the factor is 2 then the final score is (1000+10)*2 = 2020.  If the
-  # offset is not configured it defaults to 0; if the factor is not configured it defaults to 1.  The category is used
+  # the 'offset' value for the specific builder, and then multiplying by the percentage of 'factor' value. For example,
+  # if the base value is 1000, the offset is 10 and the factor is 110 then the final score is ((1000+10)/110)*100 = 1100.  If the
```

Comment:
This is sitll incorrect.  It should read `(1000+10)*110/100 = 1111`

Reply:
Yes but the code will not produce `1111` but `1100` 

```
score = new(big.Int).Div(new(big.Int).Mul(score, builderConfig.Factor), big.NewInt(100))
```